### DISCUSSION TITLE
U-he synths

### DISCUSSION THREAD

**Discussion by fabriciomrtnz** [apohl79/audiogridder#284] [Help]
Hello,

The other day I tried out moving my u-he synths (and others) to a mbp (i7 quad core) with AG. I work with logic.

The problem I came about was that the U-he synths crash very easily. For example I could not have 2 instances going at the same time. They would very easily crash and bring down AG.

CPU was around 10% usage and I had both machines connected with a direct cable at 1 gigabit. I was getting about an average of 2ms delay.

My questions is what is the recommended or the best setup. Or is my problem related to how the synths are programmed?

Should I be using a windows+vst setup or is osx+au ok/same?

Thanks!

**Comment by CaBuffalo** [comment]
Either windows or OSX set up should be fine.

Do you have the dump file / tracing file you can provide to Andreas?

**Comment by fabriciomrtnz** [comment]
Sorry no dump file.

The server seems to go into loop that I can only end by removing all plugs from logic and killing the server.

Where is the log file?

I've installed Arturia plugs and again the problem seems to be with U-he stuff.

I set the multicore flag in the plug in and it improve. But when I start loading more instances AG goes into an infinite loop.

**Reply by CaBuffalo** [reply]
In Windows task manager, find the thread of AG and right click it and them choose "create dump file"
